src/aiidalab_qe/app/result/components/viewer/structure/structure.py

```diff
@@ -9,11 +12,56 @@ def _render(self):
         if not hasattr(self, "widget"):
             structure = self._model.get_structure()
             self.widget = StructureDataViewer(structure=structure)
-            self.results_container.children = [self.widget]
+            self.widget.configuration_box.selected_index = (
+                2  # Select the Cel tab by default
+            )
+            self.table_description = ipw.HTML(
+                value="""
+                <h4 style='margin: 10px 0;'>Structure table information: Atom coordinates in Ångströms</h4>
+                <p style='margin: 5px 0; color: #555;'>
+                    You can click on a row to select an atom. Multiple atoms can be selected by clicking on additional rows. To unselect an atom, click on the selected row again.
+                </p>
+                """
+            )
```

Comment:
```suggestion
            self.table_description = ipw.HTML("""
                <h4 style='margin: 10px 0;'>
                    Structure table information: Atom coordinates in Ångströms
                </h4>
                <p style='margin: 5px 0; color: #555;'>
                    You can click on a row to select an atom. Multiple atoms 
                    can be selected by clicking on additional rows. To unselect 
                    an atom, click on the selected row again.
                </p>
            """)
```

Mostly for readability. But also, do we need to spell out angstroms? I rarely see the word 😅 If you think so, then that's fine.

Reply:
I dont know actually... i put put the symbol